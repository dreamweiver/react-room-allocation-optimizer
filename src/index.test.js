import Index from './index.js';


describe('Root component', () => {
	it('renders without crashing', () => {
	  expect(JSON.stringify(Index)).toMatchSnapshot();
	});
});