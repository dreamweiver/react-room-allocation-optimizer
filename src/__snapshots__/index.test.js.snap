// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root component renders without crashing 1`] = `"{}"`;

exports[`renders without crashing 1`] = `"{}"`;
